File: dogins/src/main/java/br/com/dogins/models/Product.java
package br.com.dogins.models;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Data

@Document("products")
public class Product {

    @Id
    private String id;

    private String productCategory;
    private String universalProductCode;
    private String productColor;
    private String productDescription;
    private List<String> productImages;
    private String brandName;
    private String productWeight;
    private String productPrice;
    private String productStock;
    private String size;
    private Integer minStock;
    private Integer amountSales;
    private String productName;
    private List<String> productColors;

}

File: MyApplication4/app/src/main/java/com/example/myapplication/models/Product.kt
package com.example.myapplication.models


data class Product(
    val id: String? = null,

    val productCategory: String,
    val universalProductCode: String,
    val productColor: String,
    val productDescription: String,
    val productImages: List<ByteArray>,
    val brandName: String,
    val productWeight: String,
    val productPrice: String,
    val productStock: String,
    val size: String,
    val minStock: String,
    val amountSales: Int,
    val productName: String,
    val productColors: List<String>
)